### PR TITLE
Fix #46 to have netcdf format for outputs in collection in Galaxy FATES tool

### LINKED ISSUE #46: Fix output format in collection for FATES Galaxy tool
- All the netCDF files are not identified as netcdf but appear as "nc" format (which does not exist in Galaxy).

### PR DESCRIPTION
**Pull request by annefou**
Thanks  Wolfgang,  Marius, and Simon for helping me out fixing this problem.
And thanks Beatriz for hosting the Galaxy papercut!

### CODE REVIEW

**Comment by bgruening** [comment]
@annefou I think you can add ftype="netcdf" to the test outputs as well.

**Comment by annefou** [comment]
> @annefou I think you can add ftype="netcdf" to the test outputs as well.

For tests we do not generate netCDF outputs (it would generate too large files for a test) because we only run a few days of simulation.

**Comment by bgruening** [comment]
> For tests we do not generate netCDF outputs (it would generate too large files for a test) because we only run a few days of simulation.

We can by now, latest Galaxy release, test for file-size without storing the netcdf in this repo. https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents --> `has_size`

**Comment by annefou** [comment]
Interesting! Let me have a look and add a new test. Many thanks for the tip.

**Comment by annefou** [comment]
@bgruening I think I need help for writing the output checks. I would like to grep for `end of main integration loop` (if we have this string anywhere in the file then it means it worked) and I had:

```
            <output name="atm_log" ftype="txt" file="atm_1m.log" compare="sim_size" delta="1000">
              <assert_contents>
                  <has_text text="end of main integration loop" />
              </assert_contents>
            </output>
```

And it works locally (with planemo and `--docker`) but it does not work online.

Now I tried to remove compare and it fails everywhere:

```
            <output name="atm_log" ftype="txt" file="atm_1m.log">
              <assert_contents>
                  <has_text text="end of main integration loop" />
              </assert_contents>
            </output>
```

And because I cannot get `atm_1m.log` I cannot really see what is wrong (if anything is wrong).

File: tools/fates-emerald/ctsm-fates.xml [hunk 1]
```diff
@@ -259,6 +254,55 @@
             <output_collection name="history_files" count="0">
             </output_collection>
         </test>
+        <test>
+            <param name="inputdata" value="inputdata_test.tar" />
+            <param name="casename" value="one_month" />
+            <param name="resolution" value="1x1_ALP1" />
+            <param name="stop_option" value="nmonths" />
+            <param name="stopn" value="1" />
+            <output name="atm_log" ftype="txt" file="atm_1m.log">
```
Comment (simonbray) [inline tools/fates-emerald/ctsm-fates.xml:263]:
```suggestion
            <output name="atm_log" ftype="txt">
```

Comment (simonbray) [reply tools/fates-emerald/ctsm-fates.xml]: You should maybe do the same for the other test outputs as well.

Comment (annefou) [reply tools/fates-emerald/ctsm-fates.xml]: Yes. I am cleaning now everything. Thank you so much!!!

**Review by bgruening — APPROVED** [review]
Cool, thanks!

**Pull request merged**